R/i18n.R

```diff
@@ -36,6 +36,13 @@ R6_i18n <- R6::R6Class(
       private$context$call("t", string, options)
     },
 
+    replace = function(text, ...) {
+      t <- function(string) {
+        self$t(string, ...)
+      }
+      vapply(text, i18n_replace1, "", t, USE.NAMES = !is.null(names(text)))
```

Comment:
Cool - I didn't know you could pass any object into vapply for the `FUN.VALUE` I thought it had to be like `character(1)` or something. I guess there is nothing special about `character()` - actually digging a little further `character()` is just creating empty strings of specified length.